src/applications/model/gaming-streaming-server.cc

```diff
@@ -312,11 +352,11 @@ void
 GamingStreamingServer::SetScalingFactor (double targetBitRate)
 {
   NS_LOG_FUNCTION (this << targetBitRate);
-  if (targetBitRate == 0)
+  if (targetBitRate == 0)  // Generate traffics based on Reference BitRate if targetBitRate not defined
```

Comment:
@mattia-lecci @salmanmohebi 
In the first PR opened by Salman some months ago, I remember we discussed that comparing a double value to 1 (i.e. == 1) is dangerous. Does this hold also in this case?

Reply:
no, in this case you voluntarily set 0.0, which is directly translated to a double number. No computations nor approximations, it should be safe as it is

Reply:
Ok, great!